### DISCUSSION TITLE
Google Maps with Place Autocomplete API

### DISCUSSION THREAD

**Discussion by kicaj** [nuxt/scripts#257] [Ideas]
It's possible to use Google Maps from Scripts to works with Place API - Autocomplete to pin pins?

**Comment by sagardudhat** [comment]
Yes, it’s possible.
you can use fitBounds method.

**Comment by harlan-zw** [comment]
Yes — the registry loads the `places` library by default, so you can use `google.maps.places.Autocomplete` directly once the script is ready. Pass `libraries` if you need to extend it. Closing — open a new discussion with a concrete API ask if you'd like first-class registry support 🙏
